test_suite/generic/test_generic.py

```diff
@@ -9,6 +12,57 @@ def test_bash_history_empty(self, host):
                 file_content_length = len(bash_history_file.content_string)
                 assert file_content_length == 0, f'{file_path} must be empty or nonexistent'
 
+    def test_console_is_redirected_to_ttys0(self, host):
+        assert host.file('/proc/cmdline').contains('console=ttyS0'),\
+            'Serial console should be redirected to ttyS0'
+
+    def test_crashkernel_is_enabled_rhel_6(self, host):
```

Comment:
I just thought maybe we could use pytest tags for tests that are for specific version. I'm not saying we should do it now, just something to think about in the future.

Reply:
Definitely, it sounds like the way to go. Let's do it in the next migration iteration!